OpenKeychain/build.gradle

```diff
@@ -135,6 +143,10 @@ android {
     dexOptions {
         preDexLibraries = false
     }
+
+    packagingOptions {
+        exclude 'LICENSE.txt'
```

Comment:
Ah, this also excludes these files from libs we use, right?
Can you explain exactly why this is exclude is added here?